packages/mermaid/src/diagrams/sankey/sankeyUtils.ts

```diff
@@ -0,0 +1,29 @@
+const prepareTextForParsing = (text: string): string => {
+  const textToParse = text
+    .replaceAll(/^[^\S\n\r]+|[^\S\n\r]+$/g, '') // remove all trailing spaces for each row
+    .replaceAll(/([\n\r])+/g, '\n') // remove empty lines duplicated
+    .trim();
+
+  return textToParse;
+};
+
+class Uid {
+  private static count = 0;
+  id: string;
+  href: string;
+
+  public static next(name: string): Uid {
+    return new Uid(name + ++Uid.count);
+  }
+
+  constructor(id: string) {
+    this.id = id;
+    this.href = `#${id}`;
+  }
+
+  toString(): string {
+    return 'url(' + this.href + ')';
+  }
+}
+
+export { Uid, prepareTextForParsing };
```

Comment:
Use direct export.

`export class Uid`

Reply:
There are 2 things to export, which export format should I prefer?
```
export function prepareTextForParsing
export class Uid
```
like that?

Reply:
Yes

Reply:
Done